client/landing/jetpack-cloud/sections/scan/history/index.jsx

```diff
@@ -3,22 +3,162 @@
+			fix:
+				'To fix this threat, Jetpack deleted the file, since it’s not a part of the original WordPress.',
+			details: 'This threat was found in the file: /htdocs/sx--a4bp.php',
+		},
+	},
+	{
+		id: 2,
+		title: 'Infected Plugin: Contact Form 7',
+		action: 'ignored',
+		detectionDate: '17 September, 2019',
+		actionDate: '24 September, 2019',
+		description: {
+			title:
+				'Unexpected file baaaaaad--file.php contains malicious code and is not part of the Plugin',
+			problem:
+				'Jetpack has detected code that is often used in web-based "shell" programs. If you believe the file(s) have been infected they need to be cleaned.',
+			fix:
+				'To fix this threat, Jetpack deleted the file, since it’s not a part of the original WordPress.',
+			details: 'This threat was found in the file: /htdocs/sx--a4bp.php',
+		},
+	},
+	{
+		id: 3,
+		title: 'Infected Plugin: Contact Form 7',
+		action: 'ignored',
+		detectionDate: '16 September, 2019',
+		actionDate: '24 September, 2019',
+		description: {
+			title:
+				'Unexpected file veryBadFile.php contains malicious code and is not part of the Plugin',
+			problem:
+				'Jetpack has detected code that is often used in web-based "shell" programs. If you believe the file(s) have been infected they need to be cleaned.',
+			fix:
+				'To fix this threat, Jetpack deleted the file, since it’s not a part of the original WordPress.',
+			details: 'This threat was found in the file: /htdocs/sx--a4bp.php',
+		},
+	},
+	{
+		id: 4,
+		title: 'Infected core file: index.php',
+		action: 'fixed',
+		detectionDate: '16 September, 2019',
+		actionDate: '24 September, 2019',
+		description: {
+			title: 'Unexpected string was found in: /htdocs/wp-admin/index.php',
+			problem:
+				'Jetpack has detected code that is often used in web-based "shell" programs. If you believe the file(s) have been infected they need to be cleaned.',
+			fix:
+				'To fix this threat, Jetpack deleted the file, since it’s not a part of the original WordPress.',
+			details: 'This threat was found in the file: /htdocs/sx--a4bp.php',
+		},
+	},
+	{
+		id: 5,
+		title: 'Infected Plugin: Classic Editor',
+		action: 'ignored',
+		detectionDate: '14 September, 2019',
+		actionDate: '14 September, 2019',
+		description: {
+			title:
+				'Unexpected file veryBadFile.php contains malicious code and is not part of the Plugin',
+			problem:
+				'Jetpack has detected code that is often used in web-based "shell" programs. If you believe the file(s) have been infected they need to be cleaned.',
+			fix:
+				'To fix this threat, Jetpack deleted the file, since it’s not a part of the original WordPress.',
+			details: 'This threat was found in the file: /htdocs/sx--a4bp.php',
+		},
+	},
+];
+
 class ScanHistoryPage extends Component {
+	state = {
+		filter: filterOptions[ 0 ],
+	};
+
+	handleOnFilterChange = filter => {
+		// @todo: should we filter in the front end?
```

Comment:
This look great. 
I think we should also add the filter to the url as the user navigates switches between the filters.
Lets do this in a different PR. 

We should also add analytics tracking to this event. This should also be done in a different PR.

Reply:
> We should also add analytics tracking to this event.

How do we track events?